### PR TITLE
GloVe

### PR DESCRIPTION
**Pull request by hujoseph99**
Took around 9 minutes to train the vectors for 92000 files using 15 iterations. The accuracy was not that good when compared against some tests, but it might not be that bad because they're different contexts. We can also fiddle around with the parameters to try increasing the accuracy

image

### CODE REVIEW

**Comment by b-saad** [comment]
lgtm

**Review by b-saad — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,7 @@
 **/Session.vim
 cnndm_acl18
 cnn_stories_tokenized/finished_files*
+cnn_stories_tokenized/1000_test
+corpus.txt
+vocab.txt
```
Comment (b-saad) [inline .gitignore:7]: shouldn't this be in the repo?

Comment (hujoseph99) [reply .gitignore:7]: I can add in the 1000_test folder but the corpus is a very large file (375mb) and the vocab doesn't help that much without the corpus. The corpus can be generated using a script I created

Comment (hujoseph99) [reply .gitignore:7]: added commit for 1000_test below

**Pull request merged**